install/helm/agones/values.yaml

```diff
@@ -45,7 +45,7 @@ agones:
   createPriorityClass: true
   priorityClassName: agones-system
   cloudProduct: "auto"
-  controller:
+  defaultValues: &defaultValues
```

Comment:
- I would definitely name this something a little less generic - they're not default across other jobs (allocation, ping service).
- But I'm also kind of vacillating now as to whether to have a common values, or to just inherit extensions from controller. Basically let's imagine a user already has `agones.controller.foobar` set - do we want `agones.extensions.foobar` also set, or do we want the override exclusive to `controller`? I see both sides.

Reply:
I think it might actually make more sense to have `extensions` inherit from `controller` unless there are plans to regularly make changes to the controller values.

Reply:
works for me, let's try it.